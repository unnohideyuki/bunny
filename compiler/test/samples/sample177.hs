x = 9223372036854775807
main = do print x
          print (x*3)
