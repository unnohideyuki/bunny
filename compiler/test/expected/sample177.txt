9223372036854775807
27670116110564327421
